doc/lsst.afw.math/Background-example.rst

```diff
@@ -0,0 +1,137 @@
+.. py:currentmodule:: lsst.afw.math
+
+.. _lsst.afw.math-BackgroundExample:
+
+###################################
+Example of lsst.afw.math.Background
+###################################
+
+Using the `Background` class; the code's in `afw/examples/estimateBackground.py <https://github.com/lsst/afw/blob/master/examples/estimateBackground.py>`_.
+
+The basic strategy is
+ - Measure the properties of the image (e.g. the mean level) -- the `Background` object
+ - `Interpolate` the `Background` to provide an estimate of the background
+ - Or generate an approximation to the `Background`, and use that to estimate the background
+
+Start by importing needed packages
+
+.. code-block:: py
+
+   import os
+   import lsst.utils
+   import lsst.afw.image as afwImage
+   import lsst.afw.math as afwMath
+
+Read an Image
+
+.. code-block:: py
+
+   def getImage():
+       imagePath = os.path.join(lsst.utils.getPackageDir("afwdata"),
+                                "DC3a-Sim", "sci", "v5-e0", "v5-e0-c011-a00.sci.fits")
```

Comment:
At least on my system, these long lines have some quite unfortunate wrapping. Maybe worth shortening them? Or do we have any other way to control the formatting?

Reply:
![screenshot 2019-01-11 at 14 31 01](https://user-images.githubusercontent.com/460854/51062923-908a0280-15ad-11e9-8fe7-403187e0e8a8.png)

Reply:
That's very strange. On *my* system, those are unwrapped:
![scrolling](https://user-images.githubusercontent.com/8508260/51064790-55d89800-15b6-11e9-9afc-7c1969157271.png)
Not sure how that can happen, unless it's either a browser default that didn't get overridden or the style is conditional on some other variable (e.g., browser width). Maybe a question for @jonathansick?

Reply:
Hmm, it looks like the stylesheet sets both `overflow` (which enables scrolling) and `word-wrap` (which is exactly what it sounds like). I wonder if there's some browser-dependent interaction between these two properties?

In any case, I think this is outside our control.